src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+|[convert](https://www.dartdocs.org/documentation/convert/latest/) | Contains encoders and decoders for converting between different data representations. Note that one of the data representations is commonly known as URL-encoded (named `percent` here). | HexDecoder, PercentDecoder |
+{:.table .table-striped .nowrap}
+
+## Specialized libraries
+
+This page doesn't include some of the more specialized libraries that are covered
+elsewhere.
+
+### Web libraries
+
+If you write web apps, check out Angular 2, a web application framework.
+Other available resources are the
+[js](https://www.dartdocs.org/documentation/js/latest) package
+for interoperability with JavaScript APIs, and the
+[dart:html](https://api.dartlang.org/stable/dart-html/dart-html-library.html)
+library for low-level HTML programming.<br>
+More information: [webdev.dartlang.org]({{site.webdev}})
+
+### Flutter libraries
+
+Use Flutter to write mobile applications.
+The core libraries distributed with the Flutter SDK are documented at
+[docs.flutter.io](http://docs.flutter.io/). To import these libraries,
+follow the instructions in [Importing libraries from
+packages](https://www.dartlang.org/tools/pub/get-started#importing-libraries-from-packages).
+Additional third-party libraries for Flutter can be found on GitHub.
+For information about how to use a shared library,
+see its README documentation on GitHub.<br>
+More information: [flutter.io]({{site.flutter}})
+
+### Dart VM libraries
+
+If you write servers or command-line applications, check out
+[dart:io](https://api.dartlang.org/stable/dart-io/dart-io-library.html)
+and related libraries.<br>
+More information: [Dart VM]({{site.dart_vm}})
+
+## Resources
+
+Use the following resources to learn more about libraries and library packages.
+
+### Importing and using libraries
+
+* [Using libraries](/guides/language/language-tour#libraries-and-visibility),
+  a section in the [language tour](/guides/language/language-tour)
+* [Importing library
+  files](/guides/libraries/create-library-packages#importing-library-files),
+  a section in [Create Library Packages](/guides/libraries/create-library-packages)
+* [Pub docs](/tools/pub), particularly
+  [Pub Package Layout Conventions](/tools/pub/package-layout)
+
+### Creating library packages
+
+* [Create Library Packages](/guides/libraries/create-library-packages)
+
+### Using specific libraries and packages
+
+* [A Tour of the Dart Libraries](/guides/libraries/library-tour)
+* Posts tagged with the
+  [Unboxing Packages](http://news.dartlang.org/search/label/Unboxing%20Packages)
+  label on on [news.dartlang.org.](http://news.dartlang.org/),
```

Comment:
I found the typo: on on!!!

The "written by NW" seems to modify news.dartlang.org, not the Unboxing Packages articles. And I think we can drop the second sentence. So maybe:

Unboxing Packages posts, written by Natalie Weizenbaum and published on news.dartlang.org.